src/main/java/com/rybka/service/exchange/BankAggregatorService.java

```diff
@@ -0,0 +1,65 @@
+package com.rybka.service.exchange;
+
+import com.rybka.constant.BankAggregatorConstants;
+import com.rybka.constant.Messages;
+import com.rybka.exception.ParameterParseException;
+import com.rybka.model.BankData;
+import coresearch.cvurl.io.model.Response;
+import coresearch.cvurl.io.request.CVurl;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Element;
+import org.springframework.stereotype.Service;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@Slf4j
+@Data
+@RequiredArgsConstructor
+public class BankAggregatorService {
+
+    private final CVurl cVurl;
+    private final String DATE_PATTERN = "yyyy-M-dd hh:mm";
```

Comment:
Why final? 
Seems spring will try to inject this value.

Reply:
We don't need provide pattern in the constructor.;

Reply:
fixed